### ISSUE TITLE
WEEK1-Introduction to Deep Learning

### ISSUE THREAD

**Issue by IvyJin0324** [IvyJin0324/Cousera-Machine-Learning#1]
**Q1：Which of the following best describes the role of AI in the expression "an AI-powered society"?**
AI is an essential ingredient in realizing tasks, in industry and in personal life.

**Q2：Which of these are reasons for Deep Learning recently taking off? (Check the three options that apply.)**
1. Deep learning has resulted in significant improvements in important applications such as online advertising, speech recognition, and image recognition. 
2. We have access to a lot more computational power. 
3. We have access to a lot more data.

**Q3：Recall the diagram of iterating over different ML ideas. Which of the stages shown in the diagram was improved with the use of a better GPU/CPU?**
Some algorithms are specifically designed to run experiments faster.

**Q4： Neural networks are good at figuring out functions relating an input xx with an output yy given enough examples. True/False?**
True.

**Q5: Which of the following depicts a Sigmoid activation function?**
![image](https://user-images.githubusercontent.com/67747655/152692431-7f9232ec-5ec7-497e-9936-958cfbc1ac3d.png)

**Q6:Images for cat recognition is an example of “structured” data because it is represented as a structured array in a computer. True/False?**
False.

**Q7: A dataset is composed of age and weight data for several people. This dataset is an example of "structured" data because it is represented as an array in a computer. True/False?**
True

**Q8: Why is an RNN (Recurrent Neural Network) used for machine translation, say translating English to French? (Check all that apply.)**
1. It can be trained as a supervised learning problem. 
2. It is applicable when the input/output is a sequence (e.g., a sequence of words). 

**Q9: Suppose the information given in the diagram is accurate. We can deduce that when using large training sets, for a model to keep improving as the amount of data for training grows, the size of the neural network must grow. True/False?.**
True

**Q10: Assuming the trends described in the figure are accurate. The performance of a NN depends only on the size of the NN. True/False?**
False
